### DISCUSSION TITLE
Pre-built OpenModelica virtual machines discontinued?

### DISCUSSION THREAD

**Discussion by pierre-haessig** [OpenModelica/OpenModelica#11630] [Q&A] [answered]
Hi,

I see on OM website's "Download" menu the fairly new Docker entry, advertised as a *"lightweight alternative to virtual machines"*. However, the [Download Virtual Machines](https://openmodelica.org/download/virtual-machine/) page is still here, but the download link to the pre-built VMs https://build.openmodelica.org/omc/builds/virtual_machines/ leads to a 404 file not found error (and a suggestion try instead https://old.build.openmodelica.org/omc/builds/virtual_machines/ which isn't better).

Is it the case that the pre-built VMs are discontinued in favor of Docker? Or is it just a temporary disruption?

(for a bit of context, I looking for the easiest options for some of my students having a Mac computer)

Best,
Pierre

**Comment by casella** [comment]
@sjoelund, @adrpo, @AnHeuermann could you comment on this?

**Comment by adrpo** [comment]
No. We moved the server data to the main openmodelica.org server but it seems we didn't copy the virtual machines properly. 
I'll have a look.

**Comment by adrpo** [comment, marked as answer]
Back online now, we were missing a symlink to the correct place:
https://build.openmodelica.org/omc/builds/virtual_machines/

**Comment by casella** [comment]
Thanks @pierre-haessig for reporting!
